fastlane/spec/fast_file_spec.rb

```diff
@@ -138,10 +146,12 @@
         ff = Fastlane::FastFile.new('./spec/fixtures/fastfiles/Fastfile1')
         ff.runner.execute(:deploy)
         expect(File.exist?('/tmp/fastlane/before_all')).to eq(true)
+        expect(File.exist?('/tmp/fastlane/before_each_deploy')).to eq(true)
         expect(File.exist?('/tmp/fastlane/deploy')).to eq(true)
         expect(File.exist?('/tmp/fastlane/test')).to eq(false)
         expect(File.exist?('/tmp/fastlane/after_all')).to eq(true)
         expect(File.read("/tmp/fastlane/after_all")).to eq("deploy")
+        expect(File.exist?('/tmp/fastlane/after_each_deploy')).to eq(true)
```

Comment:
Do those tests actually test the behavior that it is being called before every lane? We should have a separate test case just for this, that makes sure that the block is being called for each lane change, with the correct parameter.